tests/unit/babylon/src/Loading/babylon.sceneLoader.tests.ts

```diff
@@ -16,6 +16,7 @@ describe('Babylon Scene Loader', function () {
             .load(function () {
                 // Force apply promise polyfill for consistent behavior between PhantomJS, IE11, and other browsers.
                 BABYLON.PromisePolyfill.Apply(true);
+                BABYLON.Engine.audioEngine = new BABYLON.AudioEngine();

```

Comment:
@deltakosh Does this look okay? Not sure what audio engine would mean for a null engine.